R/install-github.r

```diff
@@ -243,48 +243,67 @@ parse_git_repo <- function(path) {
 }
 
 #' @export
-remote_package_name.github_remote <- function(remote, url = "https://github.com", ...) {
-
+remote_package_name.github_remote <- function(remote, ...) {
   tmp <- tempfile()
-  path <- paste(c(
-      remote$username,
-      remote$repo,
-      "raw",
-      remote$ref,
-      remote$subdir,
-      "DESCRIPTION"), collapse = "/")
-
-  req <- httr::GET(url, path = path, httr::write_disk(path = tmp))
```

Comment:
This was deliberately done using a regular http request rather than using the GitHub API because we were regularly hitting the unauthenticated GitHub API rate limits on Travis. That problem has since been ameliorated, so this is probably a reasonable change...